it/builder/builder-image/.gitignore

```diff
@@ -0,0 +1,2 @@
+/build/
```

Comment:
Is this file needed?

Reply:
This is the directory I use for built artifacts when building related CMake project. Does it matter?